samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/README.md

```diff
@@ -102,16 +102,27 @@ teddy1.jpg	teddy bear
 ```
 As you can observe, the file does not have a header row.
 
-The second step is to define the estimator pipeline. Usually, when dealing with deep neural networks, you must adapt the images to the format expected by the network. This is the reason images are resized and then transformed (mainly, pixel values are normalized across all R,G,B channels).
+The Inception model has several default parameters you need to pass in.
 
 ```csharp
+public struct ImageNetSettings
+{
+    public const int imageHeight = 224;
+    public const int imageWidth = 224;
+    public const float mean = 117;
+    public const bool channelsLast = true;
+}                
+```
+
+The second step is to define the estimator pipeline. Usually, when dealing with deep neural networks, you must adapt the images to the format expected by the network. This is the reason images are resized and then transformed (mainly, pixel values are normalized across all R,G,B channels).
+
+```csharp      
 var pipeline = mlContext.Transforms.LoadImages(outputColumnName: "input", imageFolder: imagesFolder, inputColumnName: nameof(ImageNetData.ImagePath))
-                            .Append(mlContext.Transforms.ResizeImages(outputColumnName: "input", imageWidth: ImageNetSettings.imageWidth, imageHeight: ImageNetSettings.imageHeight, inputColumnName: "input"))
-                            .Append(mlContext.Transforms.ExtractPixels(outputColumnName: "input", interleavePixelColors: ImageNetSettings.channelsLast, offsetImage: ImageNetSettings.mean))
-                            .Append(mlContext.Model.LoadTensorFlowModel(modelLocation).
-                            ScoreTensorFlowModel(outputColumnNames: new[] { "softmax2" },
-                                                inputColumnNames: new[] { "input" }, addBatchDimensionInput:true));
-                        
+    .Append(mlContext.Transforms.ResizeImages(outputColumnName: "input", imageWidth: ImageNetSettings.imageWidth, imageHeight: ImageNetSettings.imageHeight, inputColumnName: "input"))
+    .Append(mlContext.Transforms.ExtractPixels(outputColumnName: "input", interleavePixelColors: ImageNetSettings.channelsLast, offsetImage: ImageNetSettings.mean))
+    .Append(mlContext.Model.LoadTensorFlowModel(modelLocation)
+        .ScoreTensorFlowModel(outputColumnNames: new[] { "softmax2" }, inputColumnNames: new[] { "input" },
+            addBatchDimensionInput:true));
```

Comment:
Did you consider using the same indentation found in TFModelScorer.cs? (Lines 67-72 https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/ModelScorer/TFModelScorer.cs)

Reply:
I believe this indentation hides less code and is also more common throughout C# snippets, at least for the Append method calls.